mariadb_script.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Afficher un message à l'utilisateur
```

Comment:
for every script file, write in english not french

Reply:
see later gonna take time. i create à tiket to not forget